### DISCUSSION TITLE
Radiation not Dealing damage

### DISCUSSION THREAD

**Discussion by DigitalIver** [gamehog44/LethalRadiation#1] [Q&A] [answered]
After taking out aparatus i don't take damage or overtime damage only blur work.

**Comment by gamehog44** [comment]
Thanks for reaching out. Do you have damage disabled in the configuration?

**Reply by DigitalIver** [reply]
No i try both setting true and false

**Reply by gamehog44** [reply]
I would make sure both are set to true and you have the latest version installed. I have asked around and others are not experiencing the same issue. Also make sure the other damage values are not set to 0. Note that damage only occurs while you are in the factory and at the top of each hour.

**Reply by DigitalIver** [reply]
this is my setting and i am 100% sure i have turn it on, the only thing work is the blur one 
![Screenshot (1)](https://github.com/gamehog44/LethalRadiation/assets/153030687/a431678e-9b10-4ea6-9b7d-5afb1456a55c)

**Reply by DigitalIver** [reply]
if you ask there could another mod conflict with this mod then the answer is no, i have also test out only using this mod and still taking no damage, but the blur still work

**Reply by DigitalIver** [reply]
and im using r2 modmanger

**Reply by gamehog44** [reply]
Unfortunately, I’m not too sure. The last thing I could suggest is to uninstall the mod completely and reinstall.

**Reply by DigitalIver** [reply]
i just did multiple times and it still not work

**Reply by DigitalIver** [reply]
even if the damage doesn't work i still like the blur effect it give, but it would be better if their also the damage too

**Reply by DigitalIver** [reply, marked as answer]
nvm it work now after i reinstalled the mod profile

**Reply by gamehog44** [reply]
Great to hear it is working for you now! Thanks for using the mod!
